src/pages/MessageWall.js

```diff
@@ -1,11 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { app, db } from './firebase';
 
 function MessageWall() {
-  return (
-    <div>
-      Message Wall! :D
-    </div>
-  );
-}
+  const [title, setTitle] = useState();
+  const [body, setBody] = useState();
+  const [serviceArea, setServiceArea] = useState();
+  const [mentor, setMentor] = useState();
+  const [caregiver, setCaregiver] = useState();
+  const pinned = false;
+  const myTimestamp = app.firebase.firestore.Timestamp.fromDate(new Date());
+
+  const data = {
+    title,
+    body,
+    serviceArea: [serviceArea],
+    target: ['mentor', 'caregiver'],
+    pinned,
+    date: myTimestamp,
+  };
 
+  const submitData = () => {
+    db.collection('messages').doc(title).set(data);
```

Comment:
This line sets the IDs to be the title- it might be better to still use a randomly generated ID because if two posts have the same title, it might overwrite each other